dashboard/test/navigation_drawer_test.dart

```diff
@@ -72,20 +72,9 @@ void main() {
       await tester.tap(find.text(skiaPerfText));
       await tester.pump();
 
-      expect(
-        log,
-        <Matcher>[
-          isMethodCall('launch', arguments: <String, Object>{
-            'url': 'https://flutter-engine-perf.skia.org/',
-            'useSafariVC': true,
-            'useWebView': false,
-            'enableJavaScript': false,
-            'enableDomStorage': false,
-            'universalLinksOnly': false,
-            'headers': <String, String>{}
-          })
-        ],
-      );
+      expect(log[0].runtimeType, equals(MethodCall));
```

Comment:
When debugging this code, does log contain any entries (without you explicitly adding a mock method channel everywhere).

Otherwise, we'll need to look how to do this the null safe way